model/common/src/icon4py/model/common/grid/vertical.py

```diff
@@ -517,3 +540,232 @@ def get_vct_a_and_vct_b(vertical_config: VerticalGridConfig) -> tuple[fa.KField,
         if vertical_config.file_path
         else _compute_vct_a_and_vct_b(vertical_config)
     )
+
+
+def compute_SLEVE_coordinate_from_vcta_and_topography(
```

Comment:
Can you make this function either take `Field`s or `array`s, the mixture feels to me a bit strange.